### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by dambusm** [sstur/react-rte#380]
Hi, we have noticed a couple of minor accessibility issues:
- The main text input area and the block type dropdown lack labels, making them inaccessible to screen reader users. This also causes errors to be reported when using [axe-core](https://github.com/dequelabs/axe-core) or running Chrome Dev Tools' "Lighthouse" accessibility tests:
<img width="728" alt="Screenshot 2020-09-15 at 16 05 29" src="https://user-images.githubusercontent.com/12501819/93220892-61a63580-f76d-11ea-9dc9-4602c47dcac6.png">

- The buttons which can be toggled on/off (style buttons) have not been given the [switch aria role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Switch_role). This means screen reader users are unable to tell if the button is currently toggled on or off.

I will submit a PR to close this issue.
